src/main/java/nl/tudelft/lifetiles/graph/view/VertexView.java

```diff
@@ -156,12 +122,12 @@ public final double getWidth() {
 
     @Override
     protected final void layoutChildren() {
-        final double w = rectangle.getWidth();
-        final double h = rectangle.getHeight();
-        clip.setWidth(w);
-        clip.setHeight(h);
+        final double width = rectangle.getWidth();
```

Comment:
Minor inconsistency: `final` local variables. Either make all the appropriate local variables final or neither of them.